contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -358,16 +377,32 @@ contract USDTieredSTO is ISTO, ReentrancyGuard {
     * @notice fallback function - assumes ETH being invested
     */
     function () external payable {
-        buyWithETH(msg.sender);
+        buyWithETHRateLimited(msg.sender, 0);
+    }
+
+    // For backwards compatibility
+    function buyWithETH(address _beneficiary) public payable {
+        buyWithETHRateLimited(_beneficiary, 0);
+    }
+
+    function buyWithPOLY(address _beneficiary, uint256 _investedPOLY) public {
+        _buyWithTokens(_beneficiary, _investedPOLY, FundRaiseType.POLY, 0);
+    }
+
+    function buyWithUSD(address _beneficiary, uint256 _investedDAI) public validDAI {
+        _buyWithTokens(_beneficiary, _investedDAI, FundRaiseType.DAI, 0);
```

Comment:
prefer to call buyWithPOLYRateLimited() instead of _buyWithTokens() for consistency in all functions